index.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 var { validateInput } = require("./input-validation");
 
+const defaultEarthRadius = 6378137; // equatorial Earth radius
```

Comment:
We don't have a preprocesor yet :( and some folks consuming this lib are in environments that don't support let/const yet (that's the reason we're mostly using var everywhere).

Reply:
Sure, but there is const in validateNumberOfEdges already

Reply:
Interesting, maybe it's no longer a problem 👍 that must have slipped.